docs/modules/configure/pages/environment_variables.adoc

```diff
@@ -0,0 +1,30 @@
+= Environment Variables
+
+We try to let most of the things defined through Environment Variables, as https://12factor.net/config[12factor applications's config]. You'll need to configure through your hosting provider (for example in Heroku), through a gem like `figaro` or through docker-compose `env_file`.
+
+.Decidim Bulletin Board Env Vars
+|===
+|Name |Value
+
+|CORS_ORIGIN_ALLOWED
+|The domains allowed to access to the API. Use `*` for public Bulletin Boards.
+
+|DATABASE_URL
+|Connection string for the Postgres database
+
+|IDENTIFICATION_PRIVATE_KEY
+|The Bulletin Board private key, in JWT format. It should at least include the keys `kty`, `n`, `e`, `d`, `p`, `q`, `dp`, `dq` and `qi`.
+
+|RAILS_ENV
+|Environment for Ruby on Rails. If it's on your local machine for development, then it should be `development`. If it's on server it should be `production`. Please take care that this has security implications. You DON'T want to be running a production application on development mode.
+
+|RAILS_LOG_TO_STDOUT
+|If true it'll show the log contents on the shell Standard Out. It's special relevant on certain hosting providers (for example, Heroku or Docker). It's also part of https://12factor.net/logs[12factor's recomendations].
```

Comment:
```suggestion
|If true it will show the log contents on the shell Standard Out. It's especially relevant for certain hosting providers (for example, Heroku or Docker). It's also part of https://12factor.net/logs[12factor's recommendations].
```